Modules/modulesHelper.sh

```diff
@@ -0,0 +1,135 @@
+
+    # add new module to the project
+    echo 'add_subdirectory('$1')' >> CMakeLists.txt
+    
+    echo '> Module created.'
+  fi
+}
+
+# Create new task in specified module
+# \param 1 : module_name
+# \param 2 : task_name
+function create_task {
+  echo 'Creating task '$2' in module '$1'.'
+  if [ -f $1'/include/'$1'/'$2'.h' ] ; then
+    echo '> Task '$2' already exists, returning...'
+    return
+  fi
+  
+  MODULE_LC=`echo $1 | tr A-Z a-z`
+  INCLUDE_GUARD_NAME=QC_MODULE_`echo $1 | tr a-z A-Z`_`echo $2 | tr a-z A-Z`_H
+  
+  # add header
+  sed 's/'${DONOR_TASK}'/'$2'/g; s/'${DONOR_LC}'/'${MODULE_LC}'/g; s/'${DONOR}'/'${MODULE}'/g; s/'${DONOR_TASK_INCLUDE_GUARD}'/'${INCLUDE_GUARD_NAME}'/g' ${DONOR}'/include/'${DONOR}'/'${DONOR_TASK}'.h' > $1'/include/'$1'/'$2'.h' 
+  sed -i '/#endif/ i #pragma link C++ class o2::quality_control_modules::'${MODULE_LC}'::'$2'+;' $1/include/$1/LinkDef.h
+  sed -i '/set(HEADERS/ a \ \ include/'$1'/'$2'.h' $1/CMakeLists.txt
+
+  # add src
+  sed 's/'${DONOR_TASK}'/'$2'/g; s/'${DONOR_LC}'/'${MODULE_LC}'/g; s/'${DONOR}'/'${MODULE}'/g' ${DONOR}'/src/'${DONOR_TASK}'.cxx' > $1'/src/'$2'.cxx' 
+  sed -i '/set(SRCS/ a \ \ src/'$2'.cxx' $1/CMakeLists.txt
+  echo '> Task created.'
+}
+
+# Create new check in specified module
+# \param 1 : module_name
+# \param 2 : check_name
+function create_check {
+  echo 'Creating check '$2' in module '$1'.'
+  
+  if [ -f $1'/include/'$1'/'$2'.h' ] ; then
+    echo '> Check '$2' already exists, returning...'
+    return
+  fi
+  
+  MODULE_LC=`echo $1 | tr A-Z a-z`
+  INCLUDE_GUARD_NAME=QC_MODULE_`echo $1 | tr a-z A-Z`_`echo $2 | tr a-z A-Z`_H
+  
+  # add header
+  sed 's/'${DONOR_CHECK}'/'$2'/g; s/'${DONOR_LC}'/'${MODULE_LC}'/g; s/'${DONOR}'/'${MODULE}'/g; s/'${DONOR_CHECK_INCLUDE_GUARD}'/'${INCLUDE_GUARD_NAME}'/g' ${DONOR}'/include/'${DONOR}'/'${DONOR_CHECK}'.h' > $1'/include/'$1'/'$2'.h' 
+  sed -i '/#endif/ i #pragma link C++ class o2::quality_control_modules::'${MODULE_LC}'::'$2'+;' $1/include/$1/LinkDef.h
+  sed -i '/set(HEADERS/ a \ \ include/'$1'/'$2'.h' $1/CMakeLists.txt
+
+  # add src
+  sed 's/'${DONOR_CHECK}'/'$2'/g; s/'${DONOR_LC}'/'${MODULE_LC}'/g; s/'${DONOR}'/'${MODULE}'/g' ${DONOR}'/src/'${DONOR_CHECK}'.cxx' > $1'/src/'$2'.cxx' 
+  sed -i '/set(SRCS/ a \ \ src/'$2'.cxx' $1/CMakeLists.txt
+  echo '> Check created.'
+}
+
+function print_usage {
+  echo "Usage: sh modulesHelper -m MODULE_NAME [OPTION]
+
+Generate template QC module and/or tasks, checks.
+If a module with specified name already exists, new tasks and checks are inserted to the existing one.
+Please follow UpperCamelCase convention for modules', tasks' and checks' names. 
+
+Example:
+# create new module and some task
+sh modulesHelper -m MyModule -t SuperTask
+# add one task and two checks
+sh modulesHelper -m MyModule -t EvenBetterTask -c HistoUniformityCheck -c MeanTest
+
+Options:
+ -h               print this message
+ -m=MODULE_NAME   create module named MODULE_NAME or add there some task/checker
```

Comment:
Above there is no equal sign and here there is. I think that it should be removed.